### ISSUE TITLE
tabulation?

### ISSUE THREAD

**Issue by TallTed** [edlich/nosql-database.org#48]
I wonder if you've considered a tabular collection/presentation of this information?

It's hard to parse as blocks of blurb, and I see a lot of variance in how each block is worded, making comparisons difficult even if you get past the parsing challenge.

I also wonder about definitions-of-terms, as many words used here are overloaded and/or commonly misunderstood.

**Comment by edlich** [comment]
No, I did not. I thought it was easy parsable. section  + h2 and then all products in article  + h3
I do not see the difficulty. Where is this scheme broken?

The variance in the blocks itself is high as the variances in NoSQL are there. You can not get all vendors in one scheme. Impossible (we are schema-free :-)

Lastly, it would be interesting to see examples of these weird words. Perhaps we can have a word definition section at the end?!
